apisix/plugins/openid-connect.lua

```diff
@@ -328,15 +336,26 @@ function _M.rewrite(plugin_conf, ctx)
         -- Either token validation via introspection endpoint or public key is
         -- not configured, and/or token could not be extracted from the request.
 
+        local unauth_action = conf.unauth_action
+        if unauth_action ~= "auth" then
+            unauth_action = "deny"
+        end
```

Comment:
It looks like if `unauth_action` is "pass", it will be "deny" .

Reply:
"pass" is handled in the plugin, not in the oidc library. See line 354

Reply:
Get it.